catalog/opensearch/catalog-opensearch-api/src/main/java/org/codice/ddf/opensearch/source/OpenSearchParser.java

```diff
@@ -41,9 +41,10 @@ void populateSearchOptions(
       WebClient client, QueryRequest queryRequest, Subject subject, List<String> parameters);
 
   /**
-   * Fills in the OpenSearch query URL with contextual information (Note: Section 2.2 - Query: The
-   * OpenSearch specification does not define a syntax for its primary query parameter, searchTerms,
-   * but it is generally used to support simple keyword queries.)
+   * Fills in the OpenSearch query URL with the contextual information is contained in the search
+   * phrase link Map<String, String>}. (Note: Section 2.2 - Query: The OpenSearch specification does
```

Comment:
✏️ Did you mean to put "information that is contained in the search..." or maybe just "information contained in the search..."?